db/migrate/20180504140530_create_books.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBooks < ActiveRecord::Migration[5.1]
+  def change
+    create_table :books do |t|
+      t.string :genre
+      t.string :author
+      t.string :image
+      t.string :title
+      t.string :publisher
+      t.string :year
```

Comment:
If all the fields are required, neither of them should be NULL. What do you think about adding `null: false` to each one?
Assume this migration already ran in a production server, create a new one to modify all columns.